JamitLabs/JamitFoundation
# #18 improvements to action view

# Description
As described in #18 this MR adds different highlight animations to the ActionView as well as the possibility to add rounded corners.

# How to test it
1. Check out the branch and open the example app.
2. Start the example app
3. Choose "ActionView" from the list of options
4. Click on the image to see the highlight animation (Feel free to adjust the values in the view model)

# Attention
To be able to provide the example directly this MR has the feature branch as resource set in the swift package manager. After this MR is merged there must be an adjustment made here.
